Source/Syrup/Player/SyrupPlayerCharacter.h

```diff
@@ -0,0 +1,72 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Character.h"
+#include "Camera/CameraComponent.h"
+#include "SyrupPlayerCharacter.generated.h"
+
+/* \/ ===================== \/ *\
+|  \/ ASyrupPlayerCharacter \/  |
+\* \/ ===================== \/ */
+UCLASS()
+class SYRUP_API ASyrupPlayerCharacter : public ACharacter
+{
+	GENERATED_BODY()
+
+	/* ---------------- *\
+	\* \/ Components \/ */
+
+private:
+	UPROPERTY(VisibleAnywhere, Category = "Components")
+	UCameraComponent* Camera;
+
+	UPROPERTY(VisibleAnywhere, Category = "Components")
+	USceneComponent* CameraRoot;
+	
+	/* /\ Components /\ *\
+	\* ---------------- */
+
+
+public:
+	// Sets default AxisValues for this character's properties
+	ASyrupPlayerCharacter();
+
+	/* --------------- *\
+	\* \/ Variables \/ */
+private:
+	//Stores most recent X and Y input values from MoveForward() and MoveRight()
+	float recentInputX = 0.f, recentInputY = 0.f;
```

Comment:
Variable names should always be in pascal case, when in UE.  Also comment should not refer to how the variable is used internally, so no mentions to functions, just what it stores. For example: what if you added a `void Move(FVector2d)` function that also edited the variables, you wouldn't want to have to go back and change the comment for the variables.

Also I prefer(so not required), that each variable be declared in its own line with its own comment.